pints/_mcmc/_slice_doubling.py

```diff
@@ -0,0 +1,583 @@
+# -*- coding: utf-8 -*-
+#
+# Slice Sampling with Doubling MCMC Method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class SliceDoublingMCMC(pints.SingleChainMCMC):
+    """
+    *Extends:* :class:`SingleChainMCMC`
```

Comment:
This line should go right to the bottom, just before the references.

(We changed this a while ago, but apparently we failed to change it in the file you copy-pasted :D)